docs/Using-Docker.md

```diff
@@ -118,13 +119,14 @@ Notes on argument values:
 To train with a `3DBall` environment executable, the command would be:
 
 ```sh
-docker run --name 3DBallContainer.first.trial \
+docker run -it --name 3DBallContainer.first.trial \
```

Comment:
Why are we changing the Using Docker docs?

Reply:
Looks like changes from others got included in this PR, a little unsure why

Reply:
The git rebase develop introduced these changes